resources/function_help/json/dissolve_geometries

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "dissolve_geometries",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns a valid geometry or an empty geometry if the geometry could not be made valid.",
```

Comment:
this looks like the make_valid help and examples...

Reply:
Didn't had the time to change the desc and forgot about it, thanks for the reminder!